src/pretalx/cfp/templates/cfp/event/user_submission_invitation.html

```diff
@@ -0,0 +1,35 @@
+{% extends "cfp/event/base.html" %}
+{% load i18n %}
+{% load bootstrap4 %}
+{% block title %}{% trans "Your submission" %} :: {{ request.event }}{% endblock %}
+
+{% block content %}
+    <h2>
+        <a href="{{ submission.urls.user_base }}">
+            {% trans "Your submission:" %} {{ submission.title }}
+        </a>
+    </h2>
+    <p>
+        {% trans "Current state of your submission:" %}
+        {% include "cfp/event/fragment_state.html" with state=submission.state %}
+    </p>
+    <div class="alert alert-info">
+        {% blocktrans trimmed %}
+            Invite another speaker to your submission here. Instead, you can also send them
+            this link:
```

Comment:
Maybe reword this to make a clear distinction between "**We** send the link here" and "**You** send the link yourself", instead of having "you"/imperative in both cases.

Reply:
Tried to clarify